tests/unit/cli/admin/test_ceremony.py

```diff
@@ -457,6 +457,7 @@ def test_ceremony_with_flag_bootstrap_forbidden(
                     "detail": "Unauthorized.",
                 }
             ),
+            text="Unauthorized."
```

Comment:
It is related to `requests.Response`. The method `.json()` In the response is not always present. In case of error we should always parse the `Response.text`. 
It handles the errors better. I just fixed it in the ceremony.